aip_x2_launch/launch/pandar_node_container.launch.py

```diff
@@ -323,6 +325,8 @@ def add_launch_arg(name: str, default_value=None):
     add_launch_arg("input_frame", LaunchConfiguration("base_frame"))
     add_launch_arg("output_frame", LaunchConfiguration("base_frame"))
     add_launch_arg("dual_return_filter_param_file")
+    add_launch_arg("horizontal_resolution", "0.4")
+    add_launch_arg("max_distance_range", "200.0")
```

Comment:
[IMO]
It seems redundant to introduce a separate variable, max_distance_range, alongside the existing distance_range. Is it possible to use a value of distance_range instead?

Reply:
I fixed at https://github.com/tier4/aip_launcher/pull/191/commits/fa935883bae2f973dbfc51f64f7ed4bc3f896b0e